front_end/html/edit_assignment.html

```diff
@@ -247,6 +247,25 @@ <h2>Create assignment</h2>
         </p>
     </div>
 
+    <div class="shadow-box">
+        <p><strong>Restrict assignment access to the following IP addresses: </strong></br>Separate valid addresses by a comma without spaces.</p>
```

Comment:
Please indicate that this is optional, so if they leave it blank, the assignment will not be restricted by IP address.